src/components/explorer/IndividualDiseases.js

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+
+import "antd/es/table/style/css";
+import {Table} from "antd";
+
+
+import {EM_DASH} from "../../constants";
```

Comment:
lint labels this as unused

Reply:
Table is used in rendering, but will remove EM_DASH

Reply:
yeah i just meant EM_DASH! thank you